src/commands/kv/delete_bulk.rs

```diff
@@ -0,0 +1,66 @@
+extern crate base64;
+
+use cloudflare::framework::apiclient::ApiClient;
+use walkdir::WalkDir;
+
+use std::fs;
+use std::fs::metadata;
+use std::path::Path;
+
+use cloudflare::endpoints::workerskv::delete_bulk::DeleteBulk;
+use failure::bail;
+
+use crate::terminal::message;
+
+pub fn delete_bulk(namespace_id: &str, filename: &Path) -> Result<(), failure::Error> {
+    let client = super::api_client()?;
+    let account_id = super::account_id()?;
+
+    // If the provided argument for delete_bulk is a json file, parse it
+    // and delete its listed keys. If the argument is a directory, delete key-value
+    // pairs where keys are the relative pathnames of files in the directory.
+    let mut data;
+    let keys: Result<Vec<String>, failure::Error> = match metadata(filename) {
+        Ok(ref file_type) if file_type.is_file() => {
+            data = fs::read_to_string(filename)?;
+            Ok(serde_json::from_str(&data)?)
+        }
+        Ok(ref file_type) if file_type.is_dir() => parse_directory(filename),
+        Ok(_file_type) => {
+            // any other file types (namely, symlinks)
+            bail!(
+                "{} should be a file or directory, but is a symlink",
```

Comment:
nit: is symlink the only other type here? or should we interpolate `_file_type`?

Reply:
Symlink is the only other type here! I have to use a catch-all case for remaining file_type values or else the rust compiler is unhappy and thinks I am not covering all branches. I could change the wording to get rid of `but is a symlink`, and just generally point out that the provided path is not for a file or directory. What do you think?

Reply:
hmm, i think don't worry about it for now. i am wondering whether following symlinks is advisable but i'm okay with not for now, that could turn into a rabbit hole with no significant benefits.

Reply:
Coo, merging now!